src/modules/account/RadixAccount.spec.ts

```diff
@@ -15,14 +29,18 @@ describe('RadixAccount', () => {
         const actualAddressUID = account.address.getUID().toString()
 
         expect(actualAddressUID).to.equal(expectedAddressUID)
+
+        done()
     })
 
-    it('Different accounts from same seed have same address', () => {
+    it('different accounts from same seed have same address', (done) => {
```

Comment:
Why are you using 'done' here? The purpose of it is for asynchronous test cases, but there everything happens in sync, no?

Reply:
You're right here, I'll roll it back.